src/cudamatrix/cu-matrix.cc

```diff
@@ -1229,6 +1229,47 @@ void CuMatrixBase<Real>::AddMatMatElements(Real alpha,
   }
 }
 
+template<typename Real>
+void CuMatrixBase<Real>::ParametricReLU(const CuMatrixBase<Real> &src, const CuVectorBase<Real> &a, const CuVectorBase<Real> &b) {
+  KALDI_ASSERT(SameDim(*this, src));
+#if HAVE_CUDA == 1
+  if (CuDevice::Instantiate().Enabled()) {
+    Timer tim;
+
+    dim3 dimBlock(CU2DBLOCK, CU2DBLOCK);
+    dim3 dimGrid(n_blocks(src.NumCols(), CU2DBLOCK), n_blocks(src.NumRows(), CU2DBLOCK));
+
+    cuda_parametric_relu(dimGrid, dimBlock, this->data_, src.data_, this->Dim(), src.Stride(), a.data_, b.data_);
+    CU_SAFE_CALL(cudaGetLastError());
+
+    CuDevice::Instantiate().AccuProfile(__func__, tim.Elapsed());
+  } else
+#endif
+  {
+    Mat().ParametricReLU(src.Mat(), a.Vec(), b.Vec());
+  }
+}
+
+template<typename Real> 
+void CuMatrixBase<Real>::DiffParametricReLU(const CuMatrixBase<Real> &value,
```

Comment:
The indent of line 1255 is unusual. With such long function names it is fine to put one argument per line on top of each other.